meshtastic/node_runtime/content_runtime.py

```
@@ -0,0 +1,299 @@
+"""Ringtone/canned-message content cache and request/response runtimes."""
+
+import logging
+import threading
+from typing import TYPE_CHECKING, Any, Callable
```

Ruff (0.15.6)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)